docs/protocol.md

```diff
@@ -182,11 +182,15 @@ Sidetree protocol defines the following mechanisms to enable scaling, while prev
 
 #### Rate limiting
    
-   To prevent spam attack causing transaction and operation stores to grow at an unhealthy rate, rate limiting is put in place. 
+   To prevent spam attack causing transaction and operation stores to grow at an unhealthy rate, 2 types of rate limiting is put in place. 
 
-   The current implementation caps the number of operations and transactions allowed to be observed and processed. The selection logic when the caps are exceeded is the following:
+   1. writer rate limiting
+   Each writer is allowed 1 transaction per transaction time. If a writer has more than 1 transaction, the one with the lowest transaction number is chosen to be considered.
```

Comment:
To be clear, it should be "writer that require staking", smaller batch writers are not subject to this.
Also, going forward, when documentation needs to be updated:

1. we should stop updating this document, and file an issue with "documentation" tag and assign it to Daniel or Orie. and
1. Remove the relevant section in this doc.

I will let you decide if you want to it for this issue.